### PR TITLE
[Revert] Revert RestService.java

### PR DESCRIPTION
**Pull request by chovy-3012**
## Proposed changes

1. Revert RestService.java to last version (#6761 8d471007a62ffbd1d628620c4f0c44b8b6e94a8a)

```shell
git diff 8d471007a62ffbd1d628620c4f0c44b8b6e94a8a 52235a4c45757fde960d3506d85b56c25f9953bc ./extension/spark-doris-connector/src/main/java/org/apache/doris/spark/rest/RestService.java
```

2. Add a `parseBackend` method unit test

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

### CODE REVIEW

**Comment by Gabriel39** [comment]
this PR partially revert #6973

**Comment by Gabriel39** [comment]
LGTM

File: extension/spark-doris-connector/src/test/java/org/apache/doris/spark/rest/TestRestService.java [hunk 1]
```diff
@@ -293,4 +294,11 @@ public void testTabletsMapToPartition() throws Exception {
...
...ponse = "{\"href_columns\":[\"BackendId\"],\"parent_url\":\"/rest/v1/system?path=/\",\"column_names\":[\"BackendId\",\"Cluster\",\"IP\",\"HostName\",\"HeartbeatPort\",\"BePort\",\"HttpPort\",\"BrpcPort\",\"LastStartTime\",\"LastHeartbeat\",\"Alive\",\"SystemDecommissioned\",\"ClusterDecommissioned\",\"TabletNum\",\"DataUsedCapacity\",\"AvailCapacity\",\"TotalCapacity\",\"UsedPct\",\"MaxDiskUsedPct\",\"Tag\",\"ErrMsg\",\"Version\",\"Status\"],\"rows\":[{\"HttpPort\":\"8040\",\"Status\":\"{\\\"lastSuccessReportTabletsTime\\\":\\\"N/A\\\",\\\"lastStreamLoadTime\\\":-1}\",\"SystemDecommissioned\":\"false\",\"LastHeartbeat\":\"\\\\N\",\"DataUsedCapacity\":\"0.000 \",\"ErrMsg\":\"\",\"IP\":\"127.0.0.1\",\"UsedPct\":\"0.00 %\",\"__hrefPaths\":[\"/rest/v1/system?path=//backends/10002\"],\"Cluster\":\"default_cluster\",\"Alive\":\"true\",\"MaxDiskUsedPct\":\"0.00 %\",\"BrpcPort\":\"-1\",\"BePort\":\"-1\",\"ClusterDecommissioned\":\"false\",\"AvailCapacity\":\"1.000 B\",\"Version\":\"\",\"BackendId\":\"10002\",\"HeartbeatPort\":\"9050\",\"LastStartTime\":\"\\\\N\",\"TabletNum\":\"0\",\"TotalCapacity\":\"0.000 \",\"Tag\":\"{\\\"location\\\" : \\\"default\\\"}\",\"HostName\":\"localhost\"}]}";
```
Comment (chovy-3012) [inline extension/spark-doris-connector/src/test/java/org/apache/doris/spark/rest/TestRestService.java:300]: Add parseBackend unit test

File: extension/spark-doris-connector/src/test/java/org/apache/doris/spark/rest/TestRestService.java [hunk 2]
```diff
@@ -293,4 +294,23 @@ public void testTabletsMapToPartition() throws Exception {
 
         Assert.assertEquals(expected, actual);
     }
+
+    @Test
+    public void testParseBackend() throws Exception {
+        String response = "{\"href_columns\":[\"BackendId\"],\"parent_url\":\"/rest/v1/system?path=/\"," +
```
Comment (chovy-3012) [inline extension/spark-doris-connector/src/test/java/org/apache/doris/spark/rest/TestRestService.java:300]: Add a parseBackend method unit test

**Comment by Gabriel39** [comment]
cc @morningman

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**